app/ui/legacy/src/main/java/com/fsck/k9/ui/crypto/MessageCryptoCallback.java

```diff
@@ -10,6 +10,5 @@
 public interface MessageCryptoCallback {
```

Comment:
`import android.content.Intent;` could be removed

Reply:
Done.